src/sanitize-argument.ts

```diff
@@ -55,6 +56,20 @@ export async function sanitize_all(argv: any) {
     throw err;
   });
 
+  // sanitize Custom Main Page
+  if (argv.customMainPage) {
+    argv.customMainPage = argv.customMainPage.replace(/ /g,'_');
+    await axios.get(`${mwUrl}/${mwWikiPath || defaultWikiPath}${argv.customMainPage.replace(/ /g,'_')}`)
```

Comment:
Redirects should be allowed in a ios request

Reply:
@kelson42 redirects are allowed by default in axios